docs/src/configuration/app/source-operations.md

```diff
@@ -51,9 +51,39 @@ Note that these operations run in an isolated container which is not part of the
 
 Also, if multiple applications in a single project both result in a new commit, that will appear as two distinct commits in the Git history but only a single new build/deploy cycle will occur. If multiple applications define source operations with the same name, they will all be executed sequentially on each application.
 
+## Source Operations usage examples
+
+### Update a site from an upstream repository or template
+
+This example will synchronize your branch with the content from an upstream Git repository.
+
+You need to add a project-level variable named `env:UPSTREAM_REMOTE` with the Git URI of the upstream repository. That will make that repository available as a Unix environment variable in all environments, including in the Source Operations environment.
```

Comment:
```suggestion
1. Add a project-level variable named `env:UPSTREAM_REMOTE` with the Git URL of the upstream repository. That will make that repository available as a Unix environment variable in all environments, including in the Source Operations environment.

- Variable name: `env:UPSTREAM:REMOTE`
- Variable example value: `https://github.com/platformsh/platformsh-docs`
```

Reply:
For example: 
![image](https://user-images.githubusercontent.com/3729517/121389665-f3358300-c94c-11eb-8325-e43eb2342f75.png)